style/layout/_footer.scss

```diff
@@ -0,0 +1,52 @@
+.footer {
+    background: url('../../img/img_footer.svg') no-repeat;
```

Comment:
Tu mam problem jak ustawić pozycje obrazka tła, większego niż obszar tła. Chciałam zrobić to background-position: center, ale wtedy też nie jest jak w projekcie no i nie wiem.

Reply:
Możemy to też obgadać na żywo, nie nie do końca czaje o co Ci chodzi :D